Sources/ProjectDescription/DeploymentTarget.swift

```diff
@@ -5,7 +5,7 @@ import Foundation
 /// A supported minimum deployment target.
 public enum DeploymentTarget: Codable, Hashable {
     /// The minimum iOS version and the list of devices your product will support.
-    case iOS(targetVersion: String, devices: DeploymentDevice)
+    case iOS(targetVersion: String, devices: DeploymentDevice, supportsMacDesignedForIOS: Bool = true)
```

Comment:
I guess `supportsMacDesignedForIOS` is pretty self-explanatory but maybe we should still provide some really basic documentation about it and mention the default value as this is the only documentation for this that we have.